dev-lang/w2c2/w2c2-0.1~20221219.recipe

```diff
@@ -37,10 +37,13 @@ BUILD_PREREQUIRES="
 	cmd:gcc
 	cmd:make
 	"
+PATCHES="
```

Comment:
PATCHES should come after CHECKSUM_SHA256, see: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering